components/suggest/benches/benchmark_all.rs

```diff
@@ -0,0 +1,19 @@
+use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
+use suggest::benchmarks::{ingest, BenchmarkWithInput};
+
+pub fn ingest_benchmarks(c: &mut Criterion) {
+    let mut group = c.benchmark_group("ingest");
+    viaduct_reqwest::use_reqwest_backend();
+    group.sample_size(10);
```

Comment:
10 is the minimal sample size, any particular reason we want to pin it?

Reply:
The default of 100 makes the tests take a long time.  I think they would take around 150s, but I'm not totally sure because I got tired of waiting and killed them.  I'll add a note about this.